pyroomacoustics/open_sofa_interpolate.py

```diff
@@ -0,0 +1,556 @@
+import numpy as np
+import scipy
+import sofa
```

Comment:
What happens if `sofa` is not installed ? Have you added it to the list of dependencies ?

I like to keep the required dependencies to a minimum, so can we do something like
```python
try:
     import sofa
     has_sofa = True
except ImportError:
     has_sofa = False

def function_using_sofa():
    if not has_sofa:
        raise ValueError("The package 'sofa' needs to be installed to call this function. Install by doing `pip install sofa`")
````
```